app/src/main/java/com/cmput301w23t09/qrhunter/player/Player.java

```diff
@@ -151,4 +159,13 @@ public void setEmail(String email) {
 
     this.email = email;
   }
+
+  // Get all the QR code hashes associated with this player
+  public ArrayList<String> getQRCodeHashes() {
+    return qrCodeHashes;
+  }
+
+  public void setQRCodeHashes(List<String> qrcodeHashes) {
+    this.qrCodeHashes = new ArrayList<>(qrcodeHashes);
+  }
```

Comment:
Can these also have the docstring format so they are logged correctly in the Javadoc?

Reply:
ok